.github/workflows/publish-artifactory.yml

```diff
@@ -13,10 +13,13 @@
 #
 name: Publish Artifactory
 on:
+  push:
+    tags:
+      - 'v*'
   workflow_dispatch:
     inputs:
       dryrun:
-        default: true
+        default: false
```

Comment:
For the workflow_dispatch you can leave the default to true.

The on: push event won't have a inputs context so we'll have to make changes to the workflow